app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -274,15 +281,18 @@ def drs_info
       def generate_curl_config
         valid_accessions = []
         file_ids = []
-        tdr_files = {}
+        azul_files = {}
+
+        # determine if this a single-study bulk download (from download tab) or from home page search
+        search_context = params[:context] || 'global'
```

Comment:
I'm leery about a default here inflating stats one way or the other.  I'd lean towards a default of 'unknown' or maybe some logic like `(azul_files.any? || study_accessions.count > 1) ? 'global' : 'study'`

Reply:
Fair point, let me think about that a bit.

Reply:
4d0862f

Reply:
In what cases would `context` be undefined?  It strikes me as a minor analytics bug.